public/docs/_examples/toh-5/ts/app/app.module.ts

```diff
@@ -26,10 +26,10 @@ import { HeroService }  from './hero.service';
   // #docregion dashboard, hero-detail
   declarations: [
     AppComponent,
+    HeroDetailComponent,
     HeroesComponent,
-    DashboardComponent,
+    DashboardComponent
     // #enddocregion dashboard
-    HeroDetailComponent
     // #docregion dashboard
```

Comment:
Drop the #enddocregion on line 32 and this start of region (L33), since they are over the same tags.